f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_clin_clin_status.xml

```diff
@@ -7,19 +7,9 @@
         <audit>false</audit>
         <calculation><![CDATA[(function calculatedFieldValue(current) {
 
-	var now = new GlideDate();
-	var twoMonthsFromNow = new GlideDateTime();
-	twoMonthsFromNow.addDaysLocalTime(60);
-	if (current.pop_end_date < now) {
-		return 'EXPIRED';
-	}
-	else if (current.pop_start_date > now) {
-		return 'UPCOMING';
-	}
-	else if (current.pop_end_date < twoMonthsFromNow) {
-		return 'AT_RISK';
-	}
-	return 'ACTIVE';
```

Comment:
Do we want this removed or will this be done somewhere else? I thought the idea was to have this be automatically updated based on the PoP dates, but that might be scope of another ticket.

Reply:
Since the options now are:

- On-Track
- Expiring PoP
- Funding At_risk
- At-Risk
- Option Pending
- Option Exercised
- Expired
- Delinquent
I think a full ticket could be made to support all the options, if there isn't one already made.

Reply:
Fair it does seem more involved then the ticket and should be captured.